core/src/main/scala/cromwell/util/DatabaseUtil.scala

```diff
@@ -0,0 +1,22 @@
+package cromwell.util
+
+import java.sql.SQLTransientException
+
+import akka.actor.ActorSystem
+import cromwell.core.retry.{Retry, SimpleExponentialBackoff}
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+object DatabaseUtil {
```

Comment:
The problem is our `Retry.withRetry` mechanism uses `ActorSystem`s to re-schehdule futures. 
Database sub-project doesn't have akka as a dependency, which seems good to me. Other alternative would be to re-implement Retries with something else than ActorSystems using only built-in java schedulers but that seemed a lot of work for only one method using this retry anyway.